virt-v2v/cold/entrypoint.go

```diff
@@ -97,6 +97,10 @@ func main() {
 				"-io", fmt.Sprintf("vddk-thumbprint=%s", os.Getenv("V2V_fingerprint")),
 			)
 		}
+		if extraArgs := os.Getenv("V2V_extra_args"); len(extraArgs) > 0 {
```

Comment:
`checkEnvVariablesSet`

Reply:
> `checkEnvVariablesSet`

that function returns a boolean while I need the value of the environment variable